lib/lt/lcms/metadata/context.rb

```diff
@@ -9,6 +9,35 @@ module Metadata
       class Context
         attr_reader :context
 
+        NUM_RE = /\d+/
+        private_constant :NUM_RE
+
+        class << self
```

Comment:
should we call these new methods? like here - https://github.com/learningtapestry/odell-lcms/blob/master/app/services/odell/metadata_context.rb#L76
if yes, we don't need `update_grade_level_position` anymore.

Reply:
Hm. Missed that. Need to add comments to the method description if it will be called via dynamically built string.